exporters/prometheus/src/exporter_options.cc

```diff
@@ -25,7 +25,33 @@ inline const std::string GetPrometheusDefaultHttpEndpoint()
   return exists ? endpoint : kPrometheusEndpointDefault;
 }
 
-PrometheusExporterOptions::PrometheusExporterOptions() : url(GetPrometheusDefaultHttpEndpoint()) {}
+inline bool GetPrometheusPopulateOtelScope()
+{
+  constexpr char kPrometheusPopulateOtelScope[] = "PROMETHEUS_EXPORTER_POPULATE_OTEL_SCOPE";
```

Comment:
nit - should we prefix both the env variables with `OTEL_`, the naming used general in the OpenTelemetry.

Reply:
The environment variable is not defined in the spec, so it becomes language specific.

Please add a `OTEL_CPP_` prefix for both new variables, to comply with:

* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#language-specific-environment-variables